File: cpp/41A.cpp
#include <iostream>
#include <algorithm>
using namespace std;

int main() {
    string s, t;
    cin >> s >> t;

    string rev = s;
    reverse(rev.begin(), rev.end());

    if (rev == t) cout << "YES";
    else cout << "NO";

    return 0;
}

File: java/41A.java
import java.util.Scanner;

class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        String s = sc.next();
        String t = sc.next();

        String rev = new StringBuilder(s).reverse().toString();

        if (rev.equals(t)) System.out.println("YES");
        else System.out.println("NO");

        sc.close();
    }
}
